tasks/compile-example.js

```diff
@@ -60,6 +60,15 @@ module.exports = function(templateRoot, template) {
       const example = ExampleFile.fromPath(file.path);
       const nextExample = example.nextFile();
 
+      // Experimet must be injected in the first section only
+      let idx = 0;
+      const data = {
+        "sections": document.sections,
+        "experimental": function() {
+          return (idx++ == 0 && document.metadata.experiment);
```

Comment:
Can we rename `experimental` to `inExperiment`? This makes it clear that it is a boolean flag.